pkg/cmd/cmdutil/cmdutil.go

```diff
@@ -104,6 +106,7 @@ func getHandlerPairs(cmd *cobra.Command, config cfg.Config) []handlerPair {
 		{path: "/latest", handler: handlers.ListRoutesHandler},
 		{path: static.ServicePath, handler: handlers.ListRoutesHandler},
 		{path: dynamic.ServicePath, handler: handlers.ListRoutesHandler},
+		{path: userdata.ServicePath, handler: handlers.ListRoutesHandler},
```

Comment:
per my comment; don't think this is needed

Reply:
Done